tests/buildtest.cmd

```diff
@@ -87,7 +87,8 @@ if %__verbosity%==detailed (
 echo %__MsgPrefix%Commencing CoreCLR repo test build
 
 set "__BinDir=%__RootBinDir%\Product\%__BuildOS%.%__BuildArch%.%__BuildType%"
-set "__TestBinDir=%__RootBinDir%\tests\%__BuildOS%.%__BuildArch%.%__BuildType%\"
+set "__TestBinDir=%__RootBinDir%\tests"
```

Comment:
You can simplify your change a lot by defining "TestRootDir=RootBinDir\tests" and updating TestbinDir to use it. This will alleviate the need to have TestBinDirWithConfig changes.

Later you can use TestRootDir in your BaseOutputPath definition